_data/community.yml

```diff
@@ -793,3 +793,13 @@
       location: "Online & San Francisco"
       date: 2019-06-01
       link: https://zeit.co/hackathon
+-
+  id: agiletour2019
+  name: "Agile Tour Vienna"
+  events:
+    -
+      name: "Agile Tour Vienna 2019"
+      location: "FH Campus Wien"
+      date: 2019-09-20
+      link: https://agiletourvienna.at/
+	  cost: EUR 99
```

Comment:
@lanzelot1989 this is a tab and will break the build.  Please use spaces.
Also, we usually don't add events that cost unless they provide a free ticket for the community. Can you provide one?

Reply:
Fixed the tab issue, I'll see what I can do concerning a free ticket 👍

Reply:
Thank you!

Reply:
Good news!

I have confirmation that the organizer can provide one free ticket for a draw.
I suggest that only people interested in the conference take part in the draw (they should have time on 20th Sept. and be interested in the topics).

@ddprrt How would you proceed now? One of the next three Techplauscherls would be fine, I guess.

Reply:
see PM